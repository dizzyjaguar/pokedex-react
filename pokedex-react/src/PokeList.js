import React, { Component } from 'react';
import mockPokemon from './mockData.js'
import PokeItem from './PokeItem.js';

export default class PokeList extends Component {
    
    render() {
        const pokeNodes = this.props.pokedex.map(poke => <PokeItem pokeItem={poke} />)
                





      return (
        <div id='poke-list-div'>
            <ul id='poke-list'>
                {pokeNodes}
            </ul>
        </div>
      );
    }
    }