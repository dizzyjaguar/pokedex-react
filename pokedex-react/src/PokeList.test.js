import React from 'react';
import { render } from '@testing-library/react';
import PokeList from './PokeList.js';
import renderer from "react-test-renderer";

test('renders App.js correctly', () => {
  const tree = renderer
    .create(<PokeList pokedex={[]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
