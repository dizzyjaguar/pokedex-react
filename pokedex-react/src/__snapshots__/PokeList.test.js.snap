// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.js correctly 1`] = `
<div
  id="poke-list-div"
>
  <ul
    id="poke-list"
  />
</div>
`;
